universal/.agents/skills/issue-grooming/assets/issue-body-template-bug.md

```
@@ -0,0 +1,81 @@
+  - Then:
+  - **Test**:
+
+## Test Scenarios
+- [ ] Scenario 1: [short name]
+  - Setup:
+  - Action:
+  - Expected:
+  - **Test**:
+
+## Prevention
+| Failure Mode | Prevention |
```

markdownlint-cli2 (0.20.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)